### ISSUE TITLE
Feedback on Story Dashboard

### ISSUE THREAD

**Issue by gkjohn** [klpdotorg/dubdubdub#487]
1. What is the "ALL" in the top left corner of both the School and PreSchool Summary div? It has no number below it. 
2. The number of schools is all schools in WWW DB? All active schools in WWW DB? What is it? In both the summary and in each sub-section.
3. In the types of respondents, the text overruns on computer and on mobile. Also, can we not use the _ between educated_youth etc. and fix capitalisation?
4. The "What Percentage Of Schools Have These Facilities?" in Volunteer Observations has no data for any of them. Is there no data or just not in the testing DB?
5. The number of pre-schools is wrong. Again, is it meant to be all pre-schools in WWW DB? All active pre-schools in WWW DB? What is it? In both the summary and in each sub-section.
6. The number of pre-school stories is also wrong. It is the same as primary school stories. Which is incorrect. And there are no Community Surveys for pre-schools. 
7. 3. In the types of respondents, the text overruns on computer and on mobile. Also, can we not use the _ between educated_youth etc. and fix capitalisation?
8. The "What Percentage Of Pre-Schools Have These Facilities?" in Volunteer Observations has no data for any of them. Is there no data or just not in the testing DB?
9. For both schools and preschools, can we change the "ON THE DAY OF THE VISIT, THE PERCENTAGE OF SCHOOLS" to just "ON THE DAY OF THE VISIT..."

@harisibrahimkv @meghaklp @batpad 

On the whole, this is looking insanely good. Thank you.

**Comment by iambibhas** [comment]
Just a comment on point 1, maybe we should add "title" text to all the short terms like this explaining what they are.

**Comment by meghaklp** [comment]
1. I'll change this to be some sort of title, after the api changes to the branch are sealed. All indicates - all schools, but it should probably go on top and say something different.
2. @harisibrahimkv ?
3. I need to figure out the solution for this in chartist, maybe twist the labels by 45 degrees and yes the text needs formatting
4. @harisibrahimkv / @batpad can check for you.
5. 6. 7. Maybe that's what is being fixed in the api. @harisibrahimkv can respond on these.
9. 10. I'll make these changes.
11. Some api thing again @harisibrahimkv is looking into.

**Comment by meghaklp** [comment]
@iambibhas Trying to keep the titles really short. If a boundary gets selected, this thing actually says:

MLA Constituency
Bangalore North 

In 2 lines - instead of "all". Anyway, I intend to change this altogether.

**Comment by batpad** [comment]
There was a nasty bug where it was showing same summary data for schools and preschools. This has been fixed.

1. Yes, that All is a bit awkward -- it gets populated with boundary / school info when there's a filter on, but not sure what it should do / say in the All state. Easy to fix -- just need to know what to do there or if to hide it.

2. Number of schools is all active schools. Both in the top level and in each summary section. (schools and preschools respectively)

3. Will fix those label names to make them shorter.

4. Need to check up on that / wtf is going on.

5. This has been fixed. It was getting primary school summary data in both sections.

6. Yes. Should be fixed. I think it still shows 3 survey entries for preschools -- these are erroneous entries - will probably just delete them from db.

7. Yes. These were values coming from the back-end. Will coerce them to prettier values for the front-end.

8. Same as 4.

9. Sure. Let's deploy to prod then we can change the display text of questions just in the admin.

10. Right. We need to figure out the date filtering, etc. generally. @gkjohn @meghaklp Can we do a quick chat about that?

11. Fixed.

Thank you :-)

**Comment by gkjohn** [comment]
@batpad @meghaklp: The date thing isn't immediate. It's an improvement. Can be after this release also.

**Comment by batpad** [comment]
Fixed 4 / 8 (no data showing up for web facilities section)

**Comment by meghaklp** [comment]
I've fixed 1,3,9,10 in story-ui-fixes

**Comment by gkjohn** [comment]
@meghaklp On the respondents type graph, clicking the bars doesn't show the values. Please fix?

**Comment by vinayakh** [comment]
The labels can be tilted 45 degrees to make them more readable and the axes ticks should be darker. If possible show horizontal grid-lines as well to help comparison.

**Comment by meghaklp** [comment]
@gkjohn  - Yeah - that's chartist tool tips that didn't work at first go. I'll try to see how they implement it again now.

@vinayakh  - on the mobile, you mean? I've added a 90 degree rotation to that instead of 45. @gkjohn, @batpad - opinions? 

Chartist actually defaults to horizontal labels and the label text spilling over onto next lines.

**Comment by gkjohn** [comment]
@meghaklp Can you show me what the 90 degree rotation and 45 degree look like, please? Screenshot? Because on my iPhone it still is horizontal only.

**Comment by meghaklp** [comment]
![compare](https://cloud.githubusercontent.com/assets/465382/7561366/0da0599a-f7e9-11e4-9b37-5bb27bf91ac9.jpg)

**Comment by gkjohn** [comment]
Either works just fine for me. Can we change Akshara Staff to just Staff?

**Comment by gkjohn** [comment]
@batpad @vinayakh Any particular preference? Else 45.

**Comment by meghaklp** [comment]
Staff is ambiguous - Maybe just Akshara?

**Comment by gkjohn** [comment]
Also can.

**Comment by vinayakh** [comment]
@gkjohn @meghaklp Just Akshara is better. Also 45 is better than 90 for readability (don't have to tilt head to read).

**Comment by meghaklp** [comment]
Fixed tooltips on the Chartist bar charts. JQuery upgraded to 1.11.3 on story-chartist-fixes

**Comment by meghaklp** [comment]
![lollipop_collapsed](https://cloud.githubusercontent.com/assets/465382/7594125/404dd264-f8fc-11e4-8a92-a1ed52721778.png)
![lollipop_full](https://cloud.githubusercontent.com/assets/465382/7594123/3f8d5f70-f8fc-11e4-839a-c12dc4fbe103.png)

**Comment by meghaklp** [comment]
@gkjohn - above with a colour palette, I think you meant on slack.

**Comment by gkjohn** [comment]
Indeed @meghaklp. Fancy rolling it out?

**Comment by vinayakh** [comment]
@meghaklp @gkjohn The respondents graphs should be a different colour and the font should be darker. Otherwise it is not readable. See point No. 2/3 at https://medium.com/salesforce-ux/7-things-every-designer-needs-to-know-about-accessibility-64f105f0881b

**Comment by meghaklp** [comment]
Rename 
Surveys -> Community Feedback
Stories -> Field Surveys

Change images on top to reflect new headings of sections.

**Comment by meghaklp** [comment]
@vinayakh I've changed the chartist label default to be darker. But many of our colours in the prescribed colour palette have contrast issues - @batpad  - Perhaps this is a different ticket altogether. We need to check colours both from a print and contrast point of view.

Stories have been renamed to surveys @batpad  - should we change the URL alone?

**Comment by batpad** [comment]
@meghaklp yes. change the URL. But we should also add a 301 Redirect from /stories/ to the new URL, since the /stories/ link has been out there, on twitter, etc. and links should never die :)

You can look at some examples of using RedirectView in urls.py , basically would just also add something like:

    url(r'stories/$', RedirectView.as_view(url='/surveys/'))

Let know if anything doesn't make sense. Thanks.
